### PR TITLE
Currency search restyle

### LINKED ISSUE #339: Common bases border/bg
Give common bases a bg and make as the whole modal as close to the design as possible
image

https://www.figma.com/file/L8tamoiv3mTCRaypHLk1us/DXswap?node-id=4549%3A16934

### PR DESCRIPTION
**Pull request by Mi-Lan**
closes #339

### CODE REVIEW

**Comment by zamli** [comment]
I would suggest us to use the full bg background instead of the stroked bg you suggested
image
Here is the two states + the disabled/selected state

**Comment by zamli** [comment]
image
another thing i want us to check if we cna create a custom scroller similar to what mobile scroll looks like. or how apple scroll is. Is that possible?

**Comment by Mi-Lan** [comment]
Before my changes on firefox:
Screenshot 2021-09-14 at 15 20 44
After:
Screenshot 2021-09-14 at 15 37 05

I think it looks much better with these implemented....

**Comment by zamli** [comment]
Love the changes! Now we have a more narrow scroll we can make the "amount" go almost all the way to the scroll. We only need 5-6px padding

**Comment by zamli** [comment]
FLNnAXS0c7
I rather have the hover effect be like this.
This is basically removing the border from the buttons and hover color is #555a73

**Review by zamli — APPROVED** [review]

**Comment by luzzif** [comment]
Can we please add a `transition: background-color .3s ease` effect on the hover here?

File: src/components/SearchModal/styleds.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const TokenPickerItem = styled(Flex)`
   height: 56px;
   cursor: ${({ disabled }) => !disabled && 'pointer'};
   pointer-events: ${({ disabled }) => disabled && 'none'};
-  transition: background-color 0.2s ease;
+  transition: background-color 0.3s ease;
```
Comment (Mi-Lan) [inline src/components/SearchModal/styleds.tsx:45]: @luzzif I wasn't sure what you were reffering to about transiton I assumed it was here?

**Review by zamli — APPROVED** [review]

**Pull request merged**